students/EricAdams/spring_quartr_final_project/Airports.py

```diff
@@ -0,0 +1,614 @@
+        """Return the records from a csv file.  airports.csv is the default csv file.
+        files must be in the current directory
+
+        Args:
+            filename (str):  airport.csv or countries.csv.
+
+        Return:
+              Records (list):  the list returned will be in the form
+                               Records[row][column]
+        """
+        try:
+            # avoids unbound error
+            list_of_records = []
+            with open(filename, 'r') as file_obj:
+                csv_reader_obj = csv.reader(file_obj)
+                # skip the first line of the file, which lists the column names
+                list_of_records = [
+                    record for record in csv_reader_obj if record[0] != 'id']
+        except FileNotFoundError as e:
+            self.logger.error("No .csv files, error is: ", e)
+            raise
+        self.logger.info('Finished get_list_of_records_from_a_csv_file'
+                         ' returning list_of_records')
+        return list_of_records
+
+    def number_of_scheduled_or_nonscheduled_service_airports_per_country(
+            self, scheduled_or_not_scheduled):
+        """ Create a dictionary {country:xxxx} where xxxx is the number of
+        airports with scheduled service or without scheduled service.
+        airports.csv must be in the current directory.
+
+        Args:
+            scheduled_or_not_scheduled (bool):
+                                             True - return number of scheduled
+                                                service airports per country
+                                             False - return number of airports
+                                                without scheduled service.
+
+        Return:
+              number_of_airports_with_service_or_no_service_per_country (dict):
+              a dict in the form of
+              {country:numer_of_airports_with_without_service} key = (str),
+              value = (int)
+        """
+    # list_of_records_for_airport_csv[row][8] = country code
+    # list_of_records_for_airport_csv[row][11] = scheduled service
+    # list_of_records_for_airport_csv[row][1] = airport ident
+
+    # {country_name:number_of_airports_with_scheduled_service}
+
+        self.scheduled_or_not_scheduled = scheduled_or_not_scheduled
+
+        list_of_records_for_airports =\
+            self.get_list_of_records_from_a_csv_file()
+        list_of_countries_and_service = []
+        number_of_airports_with_service_or_no_service_per_country = {}
+        for row in list_of_records_for_airports:
+            list_of_countries_and_service.append([row[8], row[11]])
+        for country in list_of_countries_and_service:
+            # Sample entry of country = country['JP', 'no'] or ['JP', 'yes']
+            if self.scheduled_or_not_scheduled is True:
+                self.scheduled_or_not_scheduled = "yes"
+            if self.scheduled_or_not_scheduled is False:
```

Comment:
Can just do `if self.scheduled_or_not_scheduled:` or `if not self.schedule_or_not_scheduled`, no need for the 'is True` or `is False`